### PR TITLE
add a "Code of Conduct" 

### PR DESCRIPTION
**Pull request by bucanero**
As the PSP SDK community grows, I think it would be good to have some general guidelines and a code of conduct.

I propose this version since it's suggested by GitHub, but I'm open to tweak/adjust/change whatever we believe fits better to the PSP dev community.

### CODE REVIEW

**Comment by rofl0r** [comment]
that specific CoC you chose is well described  here https://www.devever.net/~hl/newchurch

**Comment by bucanero** [comment]
> that specific CoC you chose is well described  here https://www.devever.net/~hl/newchurch

This is not a political discussion or anything like that. 

If you want philosophical discussion then please find a place for that. 

Here we are working on software development.

**Comment by rofl0r** [comment]
> If you want philosophical discussion then please find a place for that.

second time you tell me to go away today. probably also the reason you opened this, so you can point to the document and - to quote my link  - `prosecute people for violations of the Covenant, sometimes even for thoughtcrime by project contributors uttered in forums wholly unrelated to the project in question.` in this case people being me.

> This is not a political discussion or anything like that.

the behaviour codex you want to install here is highly political, and not at all related to 
> working on software development.

**Comment by bucanero** [comment]
No, this is not a political issue at all (and not about you). It's about human respect and basic social interaction.

This PSP community is a friendly place where anyone can join and share a common interest. We are devs, yet before anything we are human beings, and it's important that everyone can feel safe around here.

In this case, the code is just setting common ground `to make participation in our
community a harassment-free experience for everyone, regardless of age, body
size, visible or invisible disability, ethnicity, sex characteristics, gender
identity and expression, level of experience, education, socio-economic status,
nationality, personal appearance, race, religion, or sexual identity
and orientation.`

With essential respect, anyone can join, and we can focus on what we like to do, which is software development.

**Comment by liclac** [comment]
Sounds good to me. We're all here to build software and have fun with our PSPs - and everyone deserves to be able to do that without fear of harassment.

**Comment by sharkwouter** [comment]
@rofl0r please refrain from posting to oftopic websites. I've removed the link from your post.

**Review by MotoLegacy — APPROVED** [review]
Looks perfect! Very comprehensive and inclusive. No reason to not merge. :)

**Review by sharkwouter — COMMENTED** [review]
I think this is a great contribution to our community, but I'm not sure if this is the right repo. We could choose to only have this on specifically the PSPDEV repo or put it on the bigger repos, maybe on the website. Do you have any ideas on that @bucanero?

**Comment by liclac** [comment]
We could even put copies of the same CoC in all of them

**Comment by bucanero** [comment]
> I think this is a great contribution to our community, but I'm not sure if this is the right repo. We could choose to only have this on specifically the PSPDEV repo or put it on the bigger repos, maybe on the website. Do you have any ideas on that @bucanero?

Yes, I think your question is valid. I think we could:
- add `CODE_OF_CONDUCT.md` to the `pspdev/pspdev` repository (assuming that's the main hub for us)
- update the `README.md` here, and on the pspsdk website, linking to the CodeOfConduct.md file 
  - note: a link can also be added to any other public repo on their local Readme.md

that way we keep a single source for the file, and we link as needed.

**Comment by sharkwouter** [comment]
That makes sense, I'll merge the other PR then.

File: README.md [hunk 1]
```diff
@@ -161,6 +161,11 @@ https://pspdev.github.io/pspsdk/
 
 You can find PSPDev Maintainers over at https://discord.gg/bePrj9W in the #psp-toolchain channel :)
 
+### Code of Conduct
+
+We're all here to build software and have fun with our PSPs, and everyone deserves to be able to do that without fear of harassment.
+
+Please follow our [Code of Conduct](CODE_OF_CONDUCT.md), and feel free to contact the PSPDev Maintainers if you think something isn't right.
```
Comment (bucanero) [inline README.md:168]:
```suggestion
Please follow our [Code of Conduct](CODE_OF_CONDUCT.md), and we encourage you to contact the PSPDev Maintainers if you think something isn't right.
```

**Comment by bucanero** [comment]
@liclac I hope you don't mind; I used your phrase when updating the `README.md`. I think it was right to the point.

@sharkwouter , as other users suggested, I think we can keep the CODE_OF_CONDUCT.md file in this repo too.

**Review by sharkwouter — APPROVED** [review]

**Pull request merged**